Selleo/repolisher
# GitHub actions labels

# Tasks:
- [x] adding action labels
- [x] updating action labels

# Added value:
- [x] Users can add and update github action labels

# PR dependency:
[1] for ex. http://github.com/PullRequest_url

# Testing scenario:
1. Go into git repository
2. run repolisher labels
3. check what labels you want to add
4. labels are created in .github/workflows directory

# Preview
Labels flow gif:

![Repolisher-labels](https://user-images.githubusercontent.com/48966657/88807458-0961a700-d1b2-11ea-86ca-8867908dd77f.gif)
